app/javascript/controllers/index.js

```diff
@@ -70,6 +70,12 @@ application.register("show-more-text", ShowMoreTextController)
 import SubmitFormController from "./submit_form_controller.js"
 application.register("submit-form", SubmitFormController)
 
+import TaskAccordionController from "./task_accordion_controller.js"
+application.register("task-accordion", TaskAccordionController)
+
+import TaskAccordionSectionController from "./task_accordion_section_controller.js"
+application.register("task-accordion-section", TaskAccordionSectionController)
+
```

Comment:
I wanted to add these in the bops-core engine but there's a jsbundling-rails issue where DHH basically says "This has been broken since webpacker and is too hard, sorry" 😭